lib/s.ml

```diff
@@ -64,6 +64,34 @@ end
 module type SANDBOX = sig
   type t
 
+  val sandbox_type : string
```

Comment:
can this just be a `val pp : t Fmt.t` instead? That would let you pretty print the abstract sandbox state, and it could simply prepend the sandbox type there as well.

Reply:
Sound very reasonable to me, thanks :)) Will also update the doc comment from above shortly

Reply:
It looks something like:
```
runc state:
 {runc_state_dir: /hello/world
  fast_sync: true
  arches: [A,B,C]}
```

Hopefully that's okay, I'm no `Format` wizard 🧙 .